docs/application-baseline.md

```diff
@@ -16,10 +16,10 @@ For optimal deployment and operational efficiency, it is important to deliver a
-| **Logging**                        | **Loki:** A log aggregation system that allows users to store, search, and analyze logs across their applications.<br><br> **Promtail:** A companion agent that efficiently gathers and sends log data to Loki, simplifying log monitoring, troubleshooting, and compliance auditing, enhancing the overall observability of the mission environment. |
-| **Security and Compliance**        | **NeuVector:** Offers container-native security, protecting applications against threats and vulnerabilities.<br><br> **Pepr:** UDS policy engine and operator for enhanced security and compliance.                                                                                                                                                  |
-| **Identity and Access Management** | **Keycloak:** A robust open-source Identity and Access Management solution, providing centralized authentication, authorization, and user management for enhanced security and control over access to mission-critical resources.                                                                                                                     |
-| **Backup and Restore**             | **Velero:** Provides backup and restore capabilities for Kubernetes clusters, ensuring data protection and disaster recovery.                                                                                                                                                                                                                         |
-| **Authorization**                  | **AuthService:** Offers centralized authorization services, managing access control and permissions within the mission environment.                                                                                                                                                                                                                   |
+| **Service Mesh**                   | **[Istio](https://istio.io/):** A powerful service mesh that provides traffic management, load balancing, security, and observability features.                                                                                                                                                                                                                       |
+| **Monitoring**                     | **[Metrics Server](https://kubernetes-sigs.github.io/metrics-server/):** Provides container resource utilization metrics API for Kubernetes clusters.<br><br>**[Prometheus](https://prometheus.io/):** Scrapes Metrics Server API and stores the data in a time-series database for insights into container health and performance.<br><br> **[Grafana](https://grafana.com/grafana/):** Provides visualization and alerting capabilities based on Prometheus's time-series database of metrics. |
```

Comment:
I think the prometheus description should also capture application metrics (not just metrics server api) since we use it to grab more than just the metrics-server? The before state might not be perfect but does capture that part for prometheus a bit, maybe can steal some wording from there.

Reply:
@mjnagel want to re-check my wording? I added a reference to application metrics, and the goal of these data points from "container health" back to "application health" as it had been.

Reply:
LGTM!